build.gradle.kts

```diff
@@ -57,6 +57,10 @@ allprojects {
         maven {
             url = uri("${project.rootDir}/../../../prebuilts/fullsdk/linux/extras/support/m2repository")
         }
+        // snapshot repository for androidx - not necessary once alpha is public
+        maven {
+            url = uri("https://androidx.dev/snapshots/builds/7352378/artifacts/repository")
```

Comment:
This is no longer necessary. This is now available on Google Maven. Replace `snapshot` with `alpha02`.

Reply:
Done